Global database and summary sheet.R

```diff
@@ -230,18 +227,18 @@ riskflags <- riskflags %>%
 
 # remove unnecessary variables
 riskflags <- riskflags %>%
-  select(-contains("_plus1"))
+  dplyr::select(-contains("_plus1"))
 
-# Alternativ combined total scores
+# Alternative combined total scores
 altflag <- globalrisk
 names <- c(
   "S_INFORM_vul_norm", "H_Oxrollback_score_norm", "H_wmo_don_alert",
   "H_Covidgrowth_casesnorm", "H_Covidgrowth_deathsnorm", "H_HIS_Score_norm", "H_INFORM_rating.Value_norm",
   "H_new_cases_smoothed_per_million_norm", "H_new_deaths_smoothed_per_million_norm",
-  "F_Proteus_Score_norm", "F_fews_crm_norm", "F_Artemis_Score_norm","F_fpv_rating", "D_WB_external_debt_distress_norm",
-  "D_IMF_debt2020.2019_norm", "M_Economic_and_Financial_score_norm", "M_GDP_IMF_2019minus2020_norm", "M_GDP_WB_2019minus2020_norm","M_macrofin_risk_norm",
-  "NH_UKMO_TOTAL.RISK.NEXT.6.MONTHS_norm", "NH_GDAC_Hazard_Score_Norm",  "H_add_death_prec_current_norm",  "H_add_death_prec_current_norm", 
-  "S_Household.risks", "S_phone_average_index_norm",  "NH_la_nina_risk", "NH_natural_acaps","Fr_FCS_FSI_Normalised", 
+  "F_Proteus_Score_norm", "F_fews_crm_norm", "F_fao_wfp_warning", "F_fpv_rating", #"D_WB_external_debt_distress_norm",
```

Comment:
Again, edits reflect changes in indicator selection. Updating which indicators the model uses.

Reply:
That said, Azavea can ignore this alternative combined total score